typing/env.ml

```diff
@@ -3075,9 +3081,17 @@ let report_lookup_error _loc env ppf = function
   | Unbound_class lid ->
       fprintf ppf "Unbound class %a" !print_longident lid;
       spellcheck ppf extract_classes env lid;
-  | Unbound_modtype lid ->
+  | Unbound_modtype lid -> begin
       fprintf ppf "Unbound module type %a" !print_longident lid;
       spellcheck ppf extract_modtypes env lid;
+      match find_namesake_module lid env with
+      | None -> ()
+      | Some _ ->
+         fprintf ppf
+           "@\nHint: There is a module called [%a] %s@?"
```

Comment:
The warning says "there is a module called [Foo]", I would (1) drop the brackets and (2) use "named" instead of "called": "there is a module named Foo".